libs/datalayout/src/main/java/com/linkedin/openhouse/datalayout/datasource/FileStat.java

```diff
@@ -10,7 +11,7 @@
 @Builder
 @NoArgsConstructor
 @AllArgsConstructor
-public final class FileStat {
+public final class FileStat implements Serializable {
```

Comment:
Meta question given you made it Serializable where are we persisting FileStat. Are we persisting it for all files in a table? If so that can be a long list.

Reply:
Thanks, I'm going to remove all serializable since we don't use default java object serialization scheme, Spark uses it's own, for strategies we use json